### DISCUSSION TITLE
LiBai 0.3.0发布计划

### DISCUSSION THREAD

**Discussion by CPFLAME** [Oneflow-Inc/libai#380] [Show and tell]
CheckList:

- [ ] Mt5的分布式训练和inference, 添加greedy search 和 beam search的generator (9月22日之前)
- [ ] swin2的精度对齐 (待定)
- [x] 添加basic inference功能 (已完成)
- [ ] dalle2的inference PR的合并 (9月22日之前)

**Comment by xiezipeng-ML** [comment]
对应pr：
Mt5的分布式训练和inference：https://github.com/Oneflow-Inc/libai/pull/382
